bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/BKTClient.kt

```diff
@@ -0,0 +1,71 @@
+package io.bucketeer.sdk.android
+
+import android.content.Context
+import io.bucketeer.sdk.android.internal.logw
+import io.bucketeer.sdk.android.internal.util.Futures
+import org.json.JSONObject
+import java.util.concurrent.Future
+
+interface BKTClient {
```

Comment:
I named the interface `BKTClient` and its implementation class `BKTClientImpl` because for me `BKTClientInteraface` and `BKTClient` as implementation class seems strange.

I can change this if you want, but in that case, users are going to call `BKTClientInterface.initialize()` and `BKTClientInterface.getInstance()`, and that looks not intuitive.

Reply:
I prefer the way you did, thanks!